Ianis/task2/ok.rb

```diff
@@ -0,0 +1,47 @@
+require "watir"
+
+class Odnoklassniki
+
+  attr_reader :browser
+  def initialize
+    @browser = Watir::Browser.new :chrome
+    browser.goto("google.com")
+    sleep(1)
```

Comment:
what if page doesn't load in 1 sec?

learn watir official documentation, there are a lot of useful helpers